### ISSUE TITLE
License file is corrupt in Next Major build. 

### ISSUE THREAD

**Issue by jonaswre** [microsoft/navcontainerhelper#2757]
**Describe the issue**
License file is corrupt in Next Major build. 
Current and Next Minor are working as expected. 

**Scripts used to create container and cause the issue**
```
AL-Go Action "Test Next Major" was used.
```

**Full output of scripts**
```
  Using license file 'c:\run\my\license.flf'
  Import License
  The license file is corrupt. Error Code: Unknown error.
  at <ScriptBlock>, C:\Run\SetupLicense.ps1: line 29
  at <ScriptBlock>, C:\Run\navstart.ps1: line 196
  at <ScriptBlock>, C:\Run\start.ps1: line 384
  at <ScriptBlock>, <No file>: line 1
```

Has the flf file type been deprecated?

**Comment by freddydk** [comment]
Yes,
In 22 only bclicense works.

**Comment by jonaswre** [comment]
Thanks.
Maybe a better error would help?
My guess, I wont be the last not to have gotten the memo. 
I've im wrong feel free to close the issue.

**Comment by freddydk** [comment]
You are also not the first:-)
I will add a warning to container if you are trying to add a .flf in 22

**Comment by freddydk** [comment]
Instead of a warning - next version will fail early if .flf is specified.

**Comment by freddydk** [comment]
Shipped in 4.0.6
